crates/biome_css_analyze/src/lint/nursery/no_css_empty_block.rs

```diff
@@ -0,0 +1,121 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_css_syntax::stmt_ext::CssBlockLike;
+use biome_deserialize_macros::Deserializable;
+use biome_rowan::AstNode;
+use serde::{Deserialize, Serialize};
+
+declare_rule! {
+    /// Disallow css empty blocks.
+    ///
+    /// This rule disallows empty block.
+    /// By default, it will allow empty blocks with comments inside.
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```css,expect_diagnostic
+    /// p {}
+    /// ```
+    ///
+    /// ```css,expect_diagnostic
+    /// .b {}
+    /// ```
+    ///
+    /// ```css,expect_diagnostic
+    /// @media print { a {} }
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```css
+    /// p {
+    ///   color: red;
+    /// }
+    /// ```
+    ///
+    /// ```css
+    /// p {
+    ///   /* foo */
+    /// }
+    /// ```
+    ///
+    /// ```css
+    /// @media print { a { color: pink; } }
+    /// ```
+    ///
+    /// ## Options
+    ///
+    /// Exclude comments from being treated as content inside of a block.
+    ///
+    /// ```json
+    /// {
+    ///     "noCssEmptyBlock": {
+    ///         "options": {
+    ///           "ignore": ["comments"]
```

Comment:
This option - coming from Styelint- isn't very... good. If by adding this option we allow comments in blocks, then why not `"options": { "allowComments": true }`? 

Also, it's really hard to understand what's the default behaviour of the rule, which we should explain in the documentation.